### PR TITLE
Added documentation page for Soma

### PR DESCRIPTION
**Pull request by ratsept**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26226

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/soma.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Soma"
+description: "Instructions on how to set up the Soma Connect within Home Assistant."
+logo: soma.png
+ha_category:
+  - Cover
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_release: 0.99
```
Comment (klaasnicolaas) [inline source/_components/soma.markdown]:
```suggestion
ha_release: "0.100"
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_components/soma.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ha_release: "0.100"
+---
+
+The Soma integration will allow users to integrate their Soma Smarthome devices into Home Assistant using the Soma Connect hub. You can build a Soma Connect yourself if you have a spare Raspberry Pi. You just need to follow the [ official instructions](https://somasmarthome.zendesk.com/hc/en-us/articles/360035521234-Install-SOMA-Connect-software-on-SOMA-Connect-Raspberry-Pi). After you have the SD card plug in the Pi and use an ethernet cable or [set up WiFI](https://somasmarthome.zendesk.com/hc/en-us/articles/360026210333-Configuring-Wi-Fi-access). Then find the IP address by checking your routers DHCP table (we will work on this step). The Connect will automatically find all your Smartshade devices in range and expose them through this integration and through Home Kit. This integration will only enumerate new shades when it is first set up. If you add shades after that you just need to restart Home Assistant or reconfigure this integration. Soma Connect will automatically discover new shades as they appear and expose them. To actually move the shades you have to first set them up according to the instructions that come with them.
+
+### Configuration
```
Comment (klaasnicolaas) [inline source/_components/soma.markdown]:
```suggestion
## Configuration
```

File: source/_components/soma.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ha_release: "0.100"
+---
+
+The Soma integration will allow users to integrate their Soma Smarthome devices into Home Assistant using the Soma Connect hub. You can build a Soma Connect yourself if you have a spare Raspberry Pi. You just need to follow the [ official instructions](https://somasmarthome.zendesk.com/hc/en-us/articles/360035521234-Install-SOMA-Connect-software-on-SOMA-Connect-Raspberry-Pi). After you have the SD card plug in the Pi and use an ethernet cable or [set up WiFI](https://somasmarthome.zendesk.com/hc/en-us/articles/360026210333-Configuring-Wi-Fi-access). Then find the IP address by checking your routers DHCP table (we will work on this step). The Connect will automatically find all your Smartshade devices in range and expose them through this integration and through Home Kit. This integration will only enumerate new shades when it is first set up. If you add shades after that you just need to restart Home Assistant or reconfigure this integration. Soma Connect will automatically discover new shades as they appear and expose them. To actually move the shades you have to first set them up according to the instructions that come with them.
```
Comment (klaasnicolaas) [inline source/_components/soma.markdown]: This is difficult to read, could you divide it into several parts?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Happy Hacktoberfest! 🎉

**Pull request merged**